pwiz_tools/Skyline/Executables/SkylineBatch/SkylineBatch.sln

```diff
@@ -89,6 +93,30 @@ Global
 		{91458586-C381-465E-9A63-0341058E0C99}.Release|x64.Build.0 = Release|Any CPU
 		{91458586-C381-465E-9A63-0341058E0C99}.Release|x86.ActiveCfg = Release|Any CPU
 		{91458586-C381-465E-9A63-0341058E0C99}.Release|x86.Build.0 = Release|Any CPU
+		{BF849CF8-25D7-4619-B001-9ED8E3771C44}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
+		{BF849CF8-25D7-4619-B001-9ED8E3771C44}.Debug|Any CPU.Build.0 = Debug|Any CPU
+		{BF849CF8-25D7-4619-B001-9ED8E3771C44}.Debug|x64.ActiveCfg = Debug|x64

```

Comment:
When a whole bunch of lines get added to the .sln file like this, it often means that you accidentally added some solution platforms that you did not mean to.
You should go to:
Build > Configuration Manager
and then choose "Edit" in the Active Solution Platform dropdown.
That will bring up a window that looks like this:
![image](https://github.com/ProteoWizard/pwiz/assets/303203/72015a2a-9183-4f3c-88fc-0856173f290d)
You should remove the "x64" and "x86" lines so that only "Any CPU" remains.